rebasehelper/tests/conftest.py

```diff
@@ -59,3 +61,15 @@ def pytest_collection_modifyitems(items):
             item.add_marker(pytest.mark.functional)
         else:
             item.add_marker(pytest.mark.standard)
+
+
+@pytest.fixture
+def mocked_spec_object(monkeypatch, spec_attributes):
+    spec = SpecFile.__new__(SpecFile)
+    for attribute, value in spec_attributes.items():
+        monkeypatch.setattr(spec, attribute, value, raising=False)
+    if 'spec_content' in spec_attributes and isinstance(spec_attributes['spec_content'], str):
+        spec.spec_content = SpecContent(spec_attributes['spec_content'])
+    if hasattr(spec, 'spec_content') and not hasattr(spec, 'tags'):
+        spec.tags = Tags(spec.spec_content, spec.spec_content)
```

Comment:
More pythonic would be to just try to assign to `spec.tags` and handle `AttributeError`.